### PR TITLE
Feat/owrt stqa 77 add picture of owner

### PR DESCRIPTION
**Pull request by 1831239**
I have made my user story work, I didn't make it possible to upload the image when a new owner is created, but once the image is in the ressources folder, you can use it to modify owners profile picture and it will be displayed. Basically, the owner object now has a string field that contains the path to the owner's profile picture in the project.

### CODE REVIEW

File: src/main/webapp/WEB-INF/jsp/owners/ownerDetails.jsp [hunk 1]
```diff
@@ -9,6 +9,10 @@
     <h2 id="ownerInformation">Owner Information</h2>
 
     <table class="table table-striped" aria-describedby="ownerInformation">
+        <tr>
+            <th id="profile_picture">Profile Picture</th>
+            <td headers="profile_picture"><img src="/STQA_pet_clinic_F2020_war/resources/images/ownersProfilePictures/${owner.profile_picture}.png" alt="owner image"></td>
```
Comment (sylveon) [inline src/main/webapp/WEB-INF/jsp/owners/ownerDetails.jsp]: Use `<spring:url>` like in #73

File: src/main/webapp/WEB-INF/jsp/owners/ownersList.jsp [hunk 2]
```diff
@@ -11,17 +11,20 @@
...
             <th scope="col" style="width: 120px">Telephone</th>
             <th scope="col" style="width: 120px">Email</th>
+            <th scope="col" style="width: 120px">Comment</th>
             <th scope="col">Pets</th>
         </tr>
         </thead>
         <tbody>
         <c:forEach items="${selections}" var="owner">
             <tr>
+                <td headers="profile_picture"><img src="/STQA_pet_clinic_F2020_war/resources/images/ownersProfilePictures/${owner.profile_picture}.png" alt="owner image"></td>
```
Comment (sylveon) [inline src/main/webapp/WEB-INF/jsp/owners/ownersList.jsp]: Idem

**Review by sylveon — CHANGES_REQUESTED** [review]

File: src/main/webapp/WEB-INF/jsp/owners/ownerDetails.jsp (hunk 1 above)
Comment (sylveon) [reply src/main/webapp/WEB-INF/jsp/owners/ownerDetails.jsp]:
```suggestion
            <spring:url value="/resources/images/ownersProfilePictures/${owner.profile_picture}.png" var="profilePictureUrl" />
            <td headers="profile_picture"><img src="${fn:escapeXml(profilePictureUrl)}" alt="owner image"></td>
```

File: src/main/webapp/WEB-INF/jsp/owners/ownersList.jsp (hunk 2 above)
Comment (sylveon) [reply src/main/webapp/WEB-INF/jsp/owners/ownersList.jsp]:
```suggestion
                <spring:url value="/resources/images/ownersProfilePictures/${owner.profile_picture}.png" var="profilePictureUrl" />
                <td headers="profile_picture"><img src="${fn:escapeXml(profilePictureUrl)}" alt="owner image"></td>
```

**Review by WilliamMeroz — DISMISSED** [review]
I have approved your pull request because Charles has already pointed out the 2 issues needed to be fixed. I would say that adding a few comments next sprint would be a great way to understand what your code is supposed to be doing.

**Review by antoheb — COMMENTED** [review]
You need to add more comments!! It is complicated to follow your changes without them. Aside from that, it is good but you will need to resolve your conflict before merging.

**Review by sylveon — APPROVED** [review]

**Review by antoheb — APPROVED** [review]
Perfect you resolved all the conflict we had in the Owner table!!

**Review by WilliamMeroz — COMMENTED** [review]
Looks good

**Pull request merged**